### ISSUE TITLE
Error when press 'g' key

### ISSUE THREAD

**Issue by Sandee022** [wsdjeg/SpaceVim#4931]
### Expected behavior.

When "g" key is pressed its showing error:-
 
Error detected while processing function SpaceVim#mapping#guide#start_by_prefix:
line    1:                                                                                                                             
E5108: Error executing lua vim/_editor.lua:0: function SpaceVim#mapping#guide#start_by_prefix[1]..script nvim_exec2() called at SpaceVi
m#mapping#guide#start_by_prefix:1: Vim(unlet):E108: No such variable: "g:leaderGuide#displayname"                                      
stack traceback:                                                                                                                       
        [C]: in function 'nvim_exec2'                                                                                                  
        vim/_editor.lua: in function 'cmd'                                                                                             
        /home/sandeep/.SpaceVim//lua/spacevim/plugin/guide.lua:136: in function 'format_displaystring'                                 
        /home/sandeep/.SpaceVim//lua/spacevim/plugin/guide.lua:270: in function 'start_parser'                                         
        /home/sandeep/.SpaceVim//lua/spacevim/plugin/guide.lua:818: in function 'start_by_prefix'                                      
        [string ":lua"]:1: in main chunk   

### The reproduce ways from Vim starting.

A bug happened!

**Comment by ledinhhuy88** [comment]
same here. 
nvim 0.10.1 homebrew
spacevim v2.4.0-dev

**Comment by victor-vb** [comment]
> same here. nvim 0.10.1 homebrew spacevim v2.4.0-dev

After downgrading to 0.9.1, no error is reported, but there is no prompt. I configured `php`. I looked at the source code and found that the latest one only supports 0.9.1. But when I opened the php file, lsp did not set it correctly for me.

**Comment by wsdjeg** [comment]
Ok, I will test it, and I can reproduce it now. thanks for your feedback.



***@***.***
 
From: victor
Date: 2024-08-25 17:26
To: SpaceVim/SpaceVim
CC: Subscribed
Subject: Re: [SpaceVim/SpaceVim] Error when press 'g' key (Issue #4931)
same here. nvim 0.10.1 homebrew spacevim v2.4.0-dev
After downgrading to 0.9.1, no error is reported, but there is no prompt. I configured php. I looked at the source code and found that the latest one only supports 0.9.1. But when I opened the php file, lsp did not set it correctly for me.
